### PR TITLE
optimize access to ssl_cert

### PR DESCRIPTION
**Pull request by arvidn**
specifically the case where there is none, avoid pessimisation of the common case

### CODE REVIEW

File: include/libtorrent/torrent_info.hpp [hunk 1]
```diff
@@ -643,7 +643,11 @@ namespace libtorrent
 			// domain in its hostname. This means the DHT and LSD
 			// features are disabled for this torrent (unless the
 			// settings allows mixing i2p peers with regular peers)
-			i2p = 4
+			i2p = 4,
+
+			// this flag is set if we found an ssl-cert field in the info
+			// dictionary
+			ssl_torrent = 8,
```
Comment (aldenml) [inline include/libtorrent/torrent_info.hpp]: I suspect this `,` at the end could be a problem, in that case try https://blog.travis-ci.com for auto cancelation of PRs (new)

Comment (arvidn) [inline include/libtorrent/torrent_info.hpp]: you mean cppcheck.py may complain about it?

Comment (arvidn) [inline include/libtorrent/torrent_info.hpp]: c++ allows this (for good reasons I think)

Comment (aldenml) [inline include/libtorrent/torrent_info.hpp]: interesting, I remember in the past getting a clang warning about this in another enum, anyway, if you have a chance, take a look at the link to enable "auto cancelation" in PRs.

**Pull request merged**